src/api/constants/export-contract-award-method/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * List of custom "export contract award method" database IDs and values.
+ */
+export const EXPORT_CONTRACT_AWARD_METHOD = {
+  OPEN_TENDER: {
+    DB_ID: 'eg9qxlqw4edxa8b5mwbybsrfp',
```

Comment:
is this the best way to do this? or do a db call and get the IDs that way?

Reply:
I believe so, we have the same approach with `COVER_PERIOD` and `TOTAL_CONTRACT_VALUE` and `DB_ID`'s.

By doing it this way, we avoid calling the API and DB in multiple scenarios (GET, POST with validation errors, maybe summary lists).

The only difference between the above mentioned constants and `EXPORT_CONTRACT_AWARD_METHOD` is that the `DB_ID` is a CUID whereas the others use incremental numbers (ideally we would update these as well)